ui/components/ui/box/box.js

```diff
@@ -169,6 +169,6 @@ Box.propTypes = {
   display: PropTypes.oneOf(Object.values(DISPLAY)),
   width: PropTypes.oneOf(Object.values(BLOCK_SIZES)),
   height: PropTypes.oneOf(Object.values(BLOCK_SIZES)),
-  backgroundColor: PropTypes.oneOf(Object.values(COLORS)),
+  backgroundColor: PropTypes.string,
```

Comment:
Color I think should be one of predefined set of colors.

Reply:
Ok. Will address in subsequent PR!